### ISSUE TITLE
Training issue

### ISSUE THREAD

**Issue by tushartalukder** [YubaoZhao/ECG-Chat#5]
Hi, I only want to use PTBXL dataset during training and evaluation. Thus, I modified the coca_train_scripts.sh to %% torchrun --nproc_per_node 1 -m open_clip.training.main --save-frequency 1 --val-frequency 5 --report-to tensorboard --warmup 10000  --batch-size 8 --lr 1e-4 --wd 0.1 --epochs 20 --workers 1 --model coca_ViT-B-32 --ptbxl-path="D:\\ECG_llm\\PTBXL\\archive\\ptb-xl-a-large-publicly-available-electrocardiography-dataset-1.0.1\\"  --lock-text --grad-clip-norm 0.5 %%
"

But it shows an error! Whenever I try to run this it shows failed to create process. 

$ bash open_clip/coca_train_scripts.sh
failed to create process.

**Comment by YubaoZhao** [comment]
Hi,

In our project, we set that MIMIC-IV-ECG dataset is necessary for training and PTB-XL is only for evaluation. If you wish to train only on PTB-XL, you can modify the `get_data` function in the file `open_clip/training/data.py` to load only PTB-XL ECG-text data. However, I think the result will be very poor because the number of training data in PTB-XL is much smaller than MIMIC-IV-ECG.
